src/actions/general.js

```diff
@@ -49,7 +49,7 @@ export function getClientConfig() {
 
 export function getLicenseConfig() {
     return bindClientFunc(
-        Client.getLicenseConfig,
+        Client4.getClientLicenseOld,
```

Comment:
We return the client config and license as a map[string]string. I have a ticket in my backlog to fix it, hence why I'm doing this here